examples/sandbox/index.ts

```diff
@@ -9,16 +7,14 @@ import * as keepkeyWebUSB from "@shapeshiftoss/hdwallet-keepkey-webusb";
 import * as keplr from "@shapeshiftoss/hdwallet-keplr";
 import * as ledgerWebHID from "@shapeshiftoss/hdwallet-ledger-webhid";
 import * as ledgerWebUSB from "@shapeshiftoss/hdwallet-ledger-webusb";
-import * as metaMask from "@shapeshiftoss/hdwallet-metamask";
```

Comment:
We weren't relying on hdwallet-metamask in web for a long, very long time already. Ditches the regular hdwallet-metamask since the multichain one superseeds it.